SPECS/kernel/config_aarch64

```diff
@@ -6916,6 +8559,8 @@ CONFIG_ARCH_HAS_UBSAN_SANITIZE_ALL=y
 # CONFIG_UBSAN is not set
 CONFIG_UBSAN_ALIGNMENT=y
 CONFIG_ARCH_HAS_DEVMEM_IS_ALLOWED=y
+CONFIG_STRICT_DEVMEM=y
+# CONFIG_IO_STRICT_DEVMEM is not set
```

Comment:
This config has been identified by the kernel config hardening checker. This config will automatically be removed when CONFIG_DEVMEM is disabled

Reply:
updated